descriptors/analysis_engines/demo.xml

```diff
@@ -142,13 +142,14 @@
       <fixedFlow>
        <node>CollectionReader</node>
        <node>ImagePreprocessor</node>
-       <node>PointCloudFilter</node>
+       <node>NormalEstimator</node>
+<!--        <node>PointCloudFilter</node>
        <node>NormalEstimator</node>
        <node>PlaneAnnotator</node>
        <node>ImageSegmentationAnnotator</node>
        <node>PointCloudClusterExtractor</node>
        <node>ClusterMerger</node>
-       <node>ResultAdvertiser</node>
+       <node>ResultAdvertiser</node> -->
```

Comment:
you could instead create a new AE that ex exemplifies the use of the data loader